LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Hoang Pham Huu <phamhuuhoang@gmail.com>
+Copyright (c) 2010-2016 Curiosity Media, Inc.
```

Comment:
I found [this](http://programmers.stackexchange.com/questions/157968/how-to-manage-a-copyright-notice-in-an-open-source-project). I'll update based on that.